lib/Executions.js

```diff
@@ -22,6 +22,20 @@ class Executions {
 
     return result
   };
+
+	async executionSummary (flowid, cb) {
```

Comment:
can you please update the indentations to match the file standards? (i.e: 2 spaces)